iphone/Maps/Categories/UIColor+MapsMeColor.h

```diff
@@ -47,6 +47,7 @@
 + (UIColor *)ratingLightGreen;
 + (UIColor *)ratingGreen;
 + (UIColor *)border;
++ (UIColor *)sharingInProgressColor;
```

Comment:
Do we really need to introduce a new color name here? Why not to use "blackHintText" or "blackSecondaryText"?